src/Activities/ServiceCollectionExtensions.cs

```diff
@@ -39,6 +39,8 @@ public static IServiceCollection AddOmexActivitySource(this IServiceCollection s
 			serviceCollection.TryAddSingleton<ActivityMetricsSender>();
 			serviceCollection.TryAddSingleton<ActivityEventSender>();
 			serviceCollection.TryAddSingleton<IActivitiesEventSender, AggregatedActivitiesEventSender>();
+			serviceCollection.TryAddSingleton<IExtraActivityBaggageDimensions>(_ => new ExtraActivityBaggageDimensions());
```

Comment:
Pay attention this these two lines. By default, `Result` `MetaData` `SubType` dimensions are collected.
Consumers can override the default dimensions by introducing another dimension set.

For example in their startup.cs, they can do:
> services.AddSingleton<IExtraActivityTagObjectsDimensions>(_ => new ExtraActivityTagObjectsDimensions(new HashSet<string> { "City", "Office" }));

Reply:
Are we loosing the ability to emit custom runtime dimensions? As some of the values of the metrics we will only know them at runtime. For example imagine a task that is executing something after a pipeline runs, where the name of the pipeline is passed and you want to have a dimension with the name of the pipeline to aggregate the metrics after, how would you do it?

Reply:
The dimension should be called something like "PipelineName" and the set of values must be finite, ideally only a handful of possible values.

Reply:
@daiiniel  No, we are not loosing it. We can register the dimension's name in through `ExtraActivityTagObjectsDimensions`. Our example of "PipelineName" can be seen as "ActivityName". The only difference is you need to register "PipelineName" to the above hashset. "ActivityName" is picked up by default